source/speech/priorities.py

```diff
@@ -1,19 +1,27 @@
 # -*- coding: UTF-8 -*-
-#A part of NonVisual Desktop Access (NVDA)
-#This file is covered by the GNU General Public License.
-#See the file COPYING for more details.
-#Copyright (C) 2006-2019 NV Access Limited
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2017-2019 NV Access Limited, Babbage B.V.
```

Comment:
I think this should stay as 2006-2019
```suggestion
# Copyright (C) 2006-2019 NV Access Limited, Babbage B.V.
```

Reply:
I made this 2017 on purpose. @jcsteh started working on speech refactor somewhere around that year, right? Speech priorities are part of speech refactor, so they weren't there in 2006.